lib/commands/l10n/import_xml_command.dart

```diff
@@ -0,0 +1,104 @@
+import 'dart:io';
+
+import 'package:alex/alex.dart';
+import 'package:alex/commands/l10n/src/l10n_command_base.dart';
+import 'package:alex/src/exception/run_exception.dart';
+import 'package:path/path.dart' as path;
+
+/// Command to import transalations from Google Play
+/// to the project's xml files.
+class ImportXmlCommand extends L10nCommandBase {
+  static const _argPath = 'path';
+
+  ImportXmlCommand()
+      : super(
+          'import_xml',
+          'Import transalations from Google Play '
+              "to the project's xml files",
```

Comment:
```suggestion
              'to the project's xml files',
```

Reply:
No, it's can't be done - because of "project's"